media/static-bundles.json

```diff
@@ -1037,19 +1037,15 @@
     },
     {
       "files": [
-        "protocol/js/protocol-modal.js",
         "js/libs/spin.min.js",
-        "js/base/send-to-device.js",
-        "js/firefox/mobile/mobile.js"
+        "js/base/send-to-device.js"
```

Comment:
@alexgibson the JS file is still here? `"js/base/send-to-device.js"`

Reply:
Is `js/firefox/mobile/mobile.js` needed to initialize the form?

Reply:
yes! added back in.